web/rootfs/etc/cont-init.d/10-config

```diff
@@ -132,6 +132,14 @@ if [[ ! -f /config/config.js ]]; then
         -e "s#// resolution:.*#resolution: 480,#" \
         /config/config.js
 
+    sed -i \
+        "s/stun:meet-jit-si-turnrelay.jitsi.net:443/${JVB_STUN_SERVERS}/g" \
```

Comment:
This variable not available in this container, please add JVB_STUN_SERVERS to corresponding service in docker-compose-custom.yml file.